fortuna/src/api.rs

```diff
@@ -137,3 +153,229 @@ impl IntoResponse for RestError {
         }
     }
 }
+
+pub fn routes(state: ApiState) -> Router<(), Body> {
+    Router::new()
+        .route("/", get(index))
+        .route("/live", get(live))
+        .route("/metrics", get(metrics))
+        .route("/ready", get(ready))
+        .route("/v1/chains", get(chain_ids))
+        .route(
+            "/v1/chains/:chain_id/revelations/:sequence",
+            get(revelation),
+        )
+        .with_state(state)
+}
+
+#[cfg(test)]
+mod test {
+    use {
+        crate::{
+            api::{
+                self,
+                ApiState,
+                BinaryEncoding,
+                Blob,
+                BlockchainState,
+                GetRandomValueResponse,
+            },
+            chain::reader::mock::MockEntropyReader,
+            state::{
+                HashChainState,
+                PebbleHashChain,
+            },
+        },
+        axum::{
+            http::StatusCode,
+            response,
+        },
+        axum_test::{
+            TestResponse,
+            TestServer,
+        },
+        ethers::prelude::Address,
+        once_cell::sync::Lazy,
+        std::sync::Arc,
+    };
+
+    const PROVIDER: Address = Address::zero();
+    const OTHER_PROVIDER: Lazy<Address> = Lazy::new(|| Address::from_low_u64_be(1));
```

Comment:
I generally think `lazy_static` is a better library/interface for this, it's sad they provide no `const` initializers from arrays. Though it seems this type is just an alias for H160 and the content of that is public, so you might be able to do:

```
const OTHER_PROVIDER: Address = Address([1u8; 20]);
```